rules/windows/discovery_posh_suspicious_api_functions.toml

```diff
@@ -111,7 +111,7 @@ event.category:process and host.os.type:windows and
     NetScheduleJobEnum or
     NetUserModalsGet
   )
-  and not user.id : ("S-1-5-18" or "S-1-5-19")
```

Comment:
As we have better visibility around PowerShell rules, I'm removing this broad exception, the rule will be tuned after the next release to fix new FPs